### ISSUE TITLE
Add option to BLOCK all social media buttons

### ISSUE THREAD

**Issue by ghost** [brave/browser-laptop#6284]
Is this something that can be implemented?

**Comment by lukemulks** [comment]
FYI - I'm rolling this issue into a parent initiative that I'm working on for social media network ads (w/in networks, and in embedded widgets and SDKs in the larger Web) - Tracking that initiative here for reference: https://github.com/brave/browser-laptop/issues/7678

**Comment by lukemulks** [comment]
We've explored the issue internally, and have included it in with an investigation into tracking and webcompat issues observed with social media buttons. 

We're currently discussing two different options for getting the end result you're looking for. Both of these options are currently rolled up into adblock UI/UX updates that are being mocked up. 

Option 1: We're working on getting the option for users to import blocklists that they have used in other extensions, directly into Brave. This would allow for the import of a mix of social media lists that you'd have control over. 

Option 2: We're also looking into including a userscript option to allow for custom scripts to be added for users that want to apply some customization to their blocking experience. 

Both of these items are currently in queue for early-phase work after our upcoming v1.0 launch. Everything in queue is currently prioritized for the v1.0 launch. 

I also wanted to let you know that as a part of the larger investigation we ran, we have dug into the Facebook ad and marketing API documentation, and have made some blocking filters to apply to audience, re-targeting and other ad tracking. I've merged those into our adblock-lists repo today: https://github.com/brave/adblock-lists/issues/21

The updates above apply to the current APIs, so there may be some instances where older/deprecated widgets and buttons are in use. I worked to make the filters flexible enough to apply with some backward-compatibility, but there may be some cases where publishers have really old buttons that they might not apply to. If observed, feel free to let us know and we'll investigate and apply filters. 

I'm going to close the issue for now, but will reference when we start pushing the features mentioned above, or if we receive new/related reports/issues.
